controller/iteration_blackbox_test.go

```diff
@@ -1129,9 +1220,10 @@ func (rest *TestIterationREST) TestUpdateIteration() {
 		payload.Data.Relationships.Parent.Data.ID = &newParentIDStr
 		payload.Data.ID = &itr3.ID
 		// when
-		_, updatedItr := test.UpdateIterationOK(t, svc.Context, svc, ctrl, itr3.ID.String(), &payload)
+		resp, updatedItr := test.UpdateIterationOK(t, svc.Context, svc, ctrl, itr3.ID.String(), &payload)
 		require.NotNil(t, updatedItr)
-		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "ok_change_parent.golden.json"), updatedItr)
+		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "ok_change_parent.iteration.golden.json"), updatedItr)
+		compareWithGoldenUUIDAgnostic(t, filepath.Join(rest.testDir, "update", "ok_change_parent.headers.golden.json"), resp)
```

Comment:
@pranavgore09 Please only store `resp.Headers()` and not the full response. The full response also contains the body and so forth.

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1783/commits/e47c6ec7c48fca69324df8ddb9991e69a122c7b6

also renamed `.error.` for golden files! thanks!

Reply:
@pranavgore09 Please revert the change from `errors` to `error`. Technically we can return more than one error at once.

Reply:
revert in https://github.com/fabric8-services/fabric8-wit/pull/1783/commits/81b436704e732a145f065b77c66fb0aeecdda9c0 :D